### DISCUSSION TITLE
How to change "defomax" and "costMode" parameters when calling snaphu_mcf unwrapper in tops_App.py

### DISCUSSION THREAD

**Discussion by hillgrey** [isce-framework/isce2#517] [General]
Dear all,
 
Which file (or files) should be modified for TOPS D-InSAR processing using ISCE2, if I would like to change the parameters such as "defomax" from 2 (or 4) into 65 that allows more discontinuity when giant phase jump occurs across the seismic fault.

Should the "costMode" be changed accordingly from SMOOTH into DEFO, too?

The situation might be considerable when giant earthquakes separate the interferograms into 2 discontinued parts and there are obvious value jumps of the deformation. Small defomax settings might lead to unreliable unwrapped results.

Thanks in advance.

**Comment by fdelgadodelapuente** [comment]
Try changing the cost mode in line 132 in install/isce/components/isceobj/TopsProc/runUnwrapSnaphu.py

**Reply by hillgrey** [reply]
Thanks for the suggestion. I will give that a try later on.

I use the snaphu_mcf unwrapper, and I am running the TOPS D-InSAR processing by modifying the file below：
isceinstall/components/isceobj/Unwrap/snaphu_mcf.py

I wonder which file is directly functional (or directly controls) with "snaphu_mcf" unwrapper. Is "isceinstall/components/isceobj/Unwrap/snaphu_mcf.py" 
totally independent from the "isce/components/isceobj/TopsProc/runUnwrapSnaphu.py"? 

Maybe, when I call "snaphu_mcf" unwrapper, the ISCE code still calls for the "isce/components/isceobj/TopsProc/runUnwrapSnaphu.py"? If so, all files that are related or linked should be edited.
